### PR TITLE
Properly propagate errors from translators

### LINKED ISSUE #76: Fizzer errors are completely ignored
Any fizzer errors that may occur are completely being ignored. For example, `ChangeColumn` discards any errors that might come from the bubbler:

```
func (f fizzer) ChangeColumn(table, name, ctype string, options Options) {
	t := Table{
		Name: table,
		Columns: []Column{
			{Name: name, ColType: ctype, Options: options},
		},
	}
	f.add(f.Bubbler.ChangeColumn(t))

	// f.add might return an error though:
	// err = f.add(f.Bubbler.ChangeColumn(t))
}
```

An error such as `can not use TEXT as a primary key` caused by `ChangeColumn` is lost. The change is not applied, and we never know why.

### PR DESCRIPTION
**Pull request by aeneasr**
Closes #76

### CODE REVIEW

**Comment by aeneasr** [comment]
Looks like PostgreSQL fails with `Error: couldn't create database pop_test: error creating PostgreSQL database pop_test: dial tcp 127.0.0.1:5432: connect: connection refused
v5.0.8`.

Not sure how to resolve that.

**Comment by aeneasr** [comment]
The tests were hanging for some reason so I `git commit --allow-empty --amend` to re-trigger the hooks. Looks like they're running now :)

File: tables.go [hunk 1]
```diff
@@ -284,21 +284,24 @@ func (f fizzer) CreateTable(name string, opts map[string]interface{}, help plush
...
 	if t.Options["timestamps"].(bool) {
 		if !t.HasColumns("created_at", "updated_at") {
-			t.Timestamp("created_at")
-			t.Timestamp("updated_at")
+			if err := t.Timestamp("created_at"); err != nil {
+				return err
+			}
+			if err := t.Timestamp("updated_at"); err != nil {
+				return err
+			}
```
Comment (stanislas-m) [inline tables.go]: Here we want to create only missing columns, that's why we didn't check for the error. `HasColumns` is a OR condition.

**Comment by stanislas-m** [comment]
Thanks!

**Pull request merged**